src/org/thoughtcrime/securesms/PinMessagesLoader.java

```diff
@@ -0,0 +1,32 @@
+package org.thoughtcrime.securesms;
```

Comment:
@g-harel the naming should be PinnedMessageLoader @erinb90 @Dan-Yoo

Reply:
you want the package to be different ?